File: plugins/OrchidPresentations/src/main/java/com/eden/orchid/presentations/PresentationsResourceSource.java
package com.eden.orchid.presentations;

import com.eden.orchid.api.OrchidContext;
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource;

import javax.inject.Inject;
import javax.inject.Singleton;

@Singleton
public class PresentationsResourceSource extends PluginResourceSource {

    @Inject
    public PresentationsResourceSource(OrchidContext context) {
        super(context, 20);
    }

}

File: plugins/OrchidPresentations/src/main/kotlin/com/eden/orchid/presentations/PresentationsResourceSource.kt
package com.eden.orchid.presentations

import com.eden.orchid.api.OrchidContext
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource

import javax.inject.Inject

class PresentationsResourceSource @Inject
constructor(context: OrchidContext) : PluginResourceSource(context, 20)
